Assets/Mirror/Runtime/Messages.cs

```diff
@@ -3,9 +3,15 @@
 
 namespace Mirror
 {
-    // This can't be an interface because users don't need to implement the
-    // serialization functions, we'll code generate it for them when they omit it.
+    public interface IMessageBase
+    {
+        void Deserialize(NetworkReader reader);
+
+        void Serialize(NetworkWriter writer);
+    }
+
     public abstract class MessageBase
+        : IMessageBase
```

Comment:
The rest of the code base specifies interfaces implemented on the same line, it'd be nice to be consistent.